db/migrate/20180104105906_create_events.rb

```diff
@@ -0,0 +1,16 @@
+class CreateEvents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :events do |t|
+      t.string :title, null: false
+      t.date :start_date
+      t.date :end_date
+      t.string :location
+      t.string :organisers
```

Comment:
Actually, I've just noticed that in the specification it says that an event can have multiple organisers and multiple organisers can be selected; while an event can have one category but multiple categories can be selected in the filters.
So I believe it's better to have categories and organisers as models and create a relationship.

https://trello.com/c/aXYh4qvg

Reply:
You are right. I need to make that change. I will create tables for the categories and the organisers and I will update the PR.